packages/ui/src/ValueView/index.tsx

```diff
@@ -82,12 +120,14 @@ export const ValueViewComponent = <SelectedContext extends Context = 'default'>(
               'border-b-2 border-dashed border-other-tonalStroke',
           )}
         >
-          <div className='min-w-[50px] shrink grow truncate' title={formattedAmount}>
-            <ValueText density={density}>{formattedAmount} </ValueText>
-          </div>
-          <div className='min-w-[50px] shrink grow truncate' title={symbol}>
-            <ValueText density={density}>{symbol}</ValueText>
+          <div className='shrink grow' title={formattedAmount}>
```

Comment:
nit; using both `shrink-0` and `shrink` in the same contexts.

Reply:
The `shrink-0` prevents the icon from shrinking, The uses of `shrink` on the other two say that is' ok to shrink.